nodejs/basic-bot/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const bodyparser = require('body-parser')
+const app = express();
+
+const PORT = process.env.PORT || 9000;
+
+app.use(bodyparser.urlencoded({ extended: false }))
+app.use(bodyparser.json())
+
+
+
+app.post('/',function(req, res){
+    
+    var test = "";

```

Comment:
This variable should be named `text`, and please using single quotes for consistency.